tweets/09-03-2022-23-7Tweet--Livestream-Kubernetes-.tweet

```diff
@@ -0,0 +1 @@
+Join the Kubernetes Office Hours this Wednesday with hosts @csantanapr and @rawkode This month is SIG-Docs edition with our guest @Divya_Mohan02https://youtu.be/KJMka1ZzVUQ
```

Comment:
```suggestion
Join the Kubernetes Office Hours this Wednesday with hosts @csantanapr and @rawkode 
This month is SIG-Docs edition with our guest @Divya_Mohan02 
https://youtu.be/KJMka1ZzVUQ
```

Reply:
The `/validate ` command to edit the initial tweet seems to be not functioning properly, so could you please close this PR and try adding your tweet content to a new issue.